Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Memefixes 8

## Changelog:
- Fixed inability to build tradepost in Wild Cape
- Lich-King champion now discards their weapon when pick ups Frostmourne. After that, discarded weapon will be transfered to another character located somewhere near Frostmourne cavern.
- `Frostfate` province holder now gets Arthas' hammer - Light's Vengeance - in 605 bookmark.
- Fixed bug where generic two-handed maces level 2-5 had support troop bonus instead of shock bonus.


## Developer changelog:
- Fixed is_electorial_trigger and is_hereditary_trigger related errors
- Fixed `limit` block in add_trait_silently_stressed_effect
- Fixed Zul holding titles before he was born
- Fixed [persistent.cpp:40]: Error: "Unexpected token: has_dlc, near line: 390" in file: "events/wc_crisis_events.txt"
- Fixed [persistent.cpp:40]: Error: "Unexpected token: show_maybe_for_humans, near line: 136" in file: "decisions/wc_economy_decisions.txt"
- Fixed [triggerimplementation.cpp:8071]: Script Assert! message: HasCharacterModifier: Unknown modifier, assert: "_pModifier", type: "none", location: " file: events/wc_on_action_events.txt"
- Fixed [triggerimplementation.cpp:15808]: Script Assert! assert: "pTitle->IsValid()", type: "none", location: " file: common/scripted_triggers/wc_scripted_triggers.txt line: 2006"
- Fixed [trigger.cpp:2193]: Unknown trigger-type: "else" at file: common/scripted_triggers/wc_scripted_triggers.txt
- Fixed [characterhistory.cpp:798]: Trying to add trait 'Cruel' despite character '63013' already having the opposite of it.
- Since you can't get plaguelands_civil_war_ongoing flag without frozen_throne_story gamerule, code don't need so much triggers with frozen_throne_story gamerule. Unnecessary ones were deleted.
- Changed elf cultural triggers in goa_main_music.txt


## How to test:

- Leave checking changes in `developer changelog` to @zumbak04 
- Check that you can build tradepost in Wild Cape after destroying goblin trade post here.
- Check that Arthas lose his weapon after he pick ups Frostmourne.
- Check that characters in `world_northrend_grizzly_hills`, `world_northrend_dragonblight` or `world_northrend_crystalsong` can get Arthas' weapon.
- Check that `frostfate` province holder gets light's vengeance in 605 bookmark.
- Check that two-handed maces level 2-5 have correct bonus.
